tests/transform/test_gene_info.py

```diff
@@ -227,6 +233,23 @@ class TestTransformGeneInfo:
         "Fail with bad data type in tep_adi_info's is_adi column",
         "Fail with bad data type in tep_adi_info's is_tep column",
     ]
+    unpiprot_input_files = {
```

Comment:
I think this section goes away since the test for the function no longer exists?

Reply:
Can you confirm if this section should go away, or stay?